first_order_model/utils.py

```diff
@@ -6,7 +6,7 @@
 from first_order_model.modules.sr_generator import SuperResolutionGenerator
 from first_order_model.modules.discriminator import MultiScaleDiscriminator
 from first_order_model.modules.keypoint_detector import KPDetector
-from swinir_wrapper import SuperResolutionModel
+#from swinir_wrapper import SuperResolutionModel
```

Comment:
please fix this back and set up your imports correctly. I can help with this issue if need be.